src/components/newcases.js

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useEffect} from 'react';
+
+function RateOfNewCases(props) {
+  const [firstDayCases, setFirstDayCases] = useState(0);
+  const [firstDayDate, setFirstDayDate] = useState(0);
+  const [secondDayCases, setSecondDayCases] = useState(0);
+  const [secondDayDate, setSecondDayDate] = useState(0);
+  const [thirdDayCases, setThirdDayCases] = useState(0);
+  const [thirdDayDate, setThirdDayDate] = useState(0);
+  const [fourthDayCases, setFourthDayCases] = useState(0);
+  const [fourthDayDate, setFourthDayDate] = useState(0);
+  const [fifthDayCases, setFifthDayCases] = useState(0);
+  const [upArrow, setUpArrow] = useState(0);
+  const [downArrow, setDownArrow] = useState(0);
```

Comment:
Why should this be in state at all? The following works and simple as well.
```js
let upArrow = String.fromCharCode(8593);
let downArrow = String.fromCharCode(8593);
```

Reply:
changed this